dokan/examples/memfs/main.rs

```diff
@@ -499,14 +496,25 @@ impl<'c, 'h: 'c> FileSystemHandler<'c, 'h> for MemFsHandler {
 		let delete_on_close = create_options & FILE_DELETE_ON_CLOSE > 0;
 		let path_info = path::split_path(&self.root, file_name)?;
 		if let Some((name, parent)) = path_info {
+			if create_options & FILE_DIRECTORY_FILE > 0 {
+				if let Some(stream_info) = &name.stream_info {
+					if !stream_info.check_default(true)? {
+						Err(STATUS_NOT_A_DIRECTORY)?;
```

Comment:
Should this return ?

Reply:
Yes, we should return an error if the directory has streams (except for the default stream ':$I30:$INDEX_ALLOCATION'). I could insert this check elsewhere, but then it would have to be inserted in multiple places, for different operations (CreateDirectory, RemoveDirectory, etc)

Reply:
Sorry I meant that I believe `return` is missing before `Err(STATUS_NOT_A_DIRECTORY)?;` ?